src/azure-cli-core/setup.py

```diff
@@ -52,7 +52,7 @@
     'humanfriendly~=10.0',
     'jmespath',
     'knack~=0.11.0',
-    'msal-extensions~=1.0.0',
+    'msal-extensions==1.2.0b1',
```

Comment:
We have to pin the version. Using `~=` will allow 1.2.0 which will contain breaking changes compared to 1.2.0b1: https://github.com/AzureAD/microsoft-authentication-extensions-for-python/pull/128#issue-2239254601

See https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release